dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -185,6 +185,7 @@ AutomatonMaximumOutputSizeReached =       automaton %1% has more outputs defined
 SystemCallFailed            =             call to %1% failed with error '%2%'
 MultipleErrors              =             %1% errors were found and displayed above
 MultipleAndHiddenErrors     =             first %1% errors displayed above
+MissingEnvironmentVariable  =             environment variable %1% is missing, you need to define it to run Dynawo
```

Comment:
I would propose "mandatory environment variable %1% is not defined"

Reply:
ok done